modules/TypeDefinitions.js

```diff
@@ -20,7 +20,11 @@ export type RouteProps = {
   children?: (props: ContextRouter) => React$Element<any>,
   path: string,
   exact?: boolean,
-  strict?: boolean
+  strict?: boolean,
+  state?: NavigationState<{
```

Comment:
OK! I was about to use `Object`in the 1st place, but it sound pretty awkward to me